rfcs/0004-leader-rotation.md

```diff
@@ -1,104 +1,29 @@
-3. The schedule is applicable for `M` voting rounds
-
-Leader's transmit for a count of `T` PoH ticks.  When `T` is reached all the validators should switch to the next scheduled leader.  To schedule leaders, the supermajority + `M` nodes are shuffled using the above calculated random seed.
-
-All `T` ticks must be observed from the current leader for that part of PoH to be accepted by the network.  If `T` ticks (and any intervening transactions) are not observed, the network optimistically fills in the `T` ticks, and continues with PoH from the next leader.
-
-## Partitions, Forks
-
-Forks can arise at PoH tick counts that correspond to leader rotations, because leader nodes may or may not have observed the previous leader's data.  These empty ticks are generated by all nodes in the network at a network-specified rate for hashes/per/tick `Z`.
-
-There are only two possible versions of the PoH during a voting period: PoH with `T` ticks and entries generated by the current leader, or PoH with just ticks.  The "just ticks" version of the PoH can be thought of as a virtual ledger, one that all nodes in the network can derive from the last tick in the previous period.
-
-Validators can ignore forks at other points (e.g. from the wrong leader), or slash the leader responsible for the fork.
-
-Validators vote on the longest chain that contains their previous vote, or a longer chain if the lockout on their previous vote has expired.
-
-
-#### Validator's View
-
-##### Time Progression
-The diagram below represents a validator's view of the PoH stream with possible forks over time.  L1, L2, etc. are leader periods, and `E`s represent entries from that leader during that leader's period.  The 'x's represent ticks only, and time flows downwards in the diagram.
-
-
-```
-time  +----+                                          validator action
-|     | L1 |                 E(L1)
-|     |----|            /             \                vote(E(L2))
-|     | L2 |        E(L2)               x
-|     |----|        /    \           /     \           vote(E(L2))
-|     | L3 |    E(L3)      x       E(L3)'    x
-|     |----|    /  \     /   \     /  \     /  \       slash(L3)
-|     | L4 |  x     x  E(L4)  x   x    x   x    x
-V     |----|  |     |  |      |   |    |   |    |      vote(E(L4))
-V     | L5 |  xx   xx  xx   E(L5) xx   xx  xx   xx
-V     +----+                                           hang on to E(L4) and E(L5) for more...
+The seed that is selected is predictable but unbiasable.  There is no grinding attack to influence its outcome.  The set of **staked accounts** and their leader identities is computed over a large period, which is our approach to censorship resistance of the staking set.  If at least 1 leader in the schedule is not censoring staking transactions then over a long period of time that leader can ensure that the set of active nodes is not censored.
```

Comment:
"our approach to censorship resistance" -> "is used to resist censorship". Also, how? Alternatively say that's out of the scope of this document.

Also, typos in the last sentence. Two missing commas and "is" should be "are".

That last sentence should either be dropped (out of scope) or expanded on a bunch.